sql/wsrep_server_state.cc

```diff
@@ -50,14 +50,15 @@ void Wsrep_server_state::init_once(const std::string &name,
                                    const std::string &address,
                                    const std::string &working_dir,
                                    const wsrep::gtid &initial_position,
-                                   int max_protocol_version) {
+                                   const wsrep_version &max_protocol_version) {
```

Comment:
I think there is no advantage in passing it as the referece

Reply:
Agree. Will update it.